userspace/falco/app/actions/configure_interesting_sets.cpp

```diff
@@ -89,11 +89,9 @@ static void select_event_set(falco::app::state& s, const libsinsp::events::set<p
 	auto user_positive_sc_set = libsinsp::events::names_to_sc_set(user_positive_names);
 	auto user_negative_sc_set = libsinsp::events::names_to_sc_set(user_negative_names);
 
-	if (!user_positive_sc_set.empty() || s.config->m_base_syscalls_repair)
+	if (!user_positive_sc_set.empty())
```

Comment:
note: we were wrongly accepting totally-empty positive set, which we didn't allow before and I'm a bit contrary on.

Reply:
What would be an alternative to allow bypassing default sinsp state enforcement, but use .repair option, but maybe still want one negative syscall?

Either way you can always just add execve in there and that way bypass, so I would always find a way to get the behavior I want :)

Reply:
My points are:
- We need a way for users to satisfy the case "all default sinsp set syscalls, except the negated ones", which as of today we interpret the empty positive set for. If you want to really have an empty custom positive set, what is your proposal to support the above? We do the same kind of reasoning for enabled event sources, for example
- I'm not sure that explicitly allowing user to pass-in totally-empty base sets is the best UX, and as you said you still have means to circumvent it by simply defining a syscall as positive and negative at the same time. I'm not strong on this, but what is the concrete use case that you foresee on having a totally-empty positive base set?